src/components/ViewArticles.jsx

```diff
@@ -1,9 +1,33 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
 function ViewArticles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios.get('http://localhost:9090/api/articles')
```

Comment:
this should be extracted out to its own file in a utils folder

Reply:
Done!